src/SettingsView/SettingsView.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {connect} from 'react-refetch';
+import ConnectionComponent from './../Utils/ConnectionComponent.js';
+import EventType from './EventType';
+import SearchBar from './../Utils/SearchBar';
+
+
+class SettingsView extends ConnectionComponent {
+
+	render() {
+		const connectionIncomplete = super.render(this.props.eventTypes);
+		if(connectionIncomplete) {
+			return connectionIncomplete;
+		}
+		return (
+			<div>
+				<SearchBar/>
+				{this.props.eventTypes.value.map((eventType) => {
+					return(<EventType 
+						eventType={eventType} 
+						key={eventType.Id}
+						deleteEventType={this.props.deleteEventType} />);
+				})}
+			</div>
+		);
+	}
+}
+
+export default connect.defaults({fetch: ConnectionComponent.switchFetch})(props => ({
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "props". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)